rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/TaskController.java

```diff
@@ -198,8 +198,31 @@ public ResponseEntity<TaskRepresentationModel> claimTask(
       throws TaskNotFoundException, InvalidStateException, InvalidOwnerException,
           NotAuthorizedException {
     // TODO verify user
-    taskService.claim(taskId);
-    Task updatedTask = taskService.getTask(taskId);
+    Task updatedTask = taskService.claim(taskId);
+    return ResponseEntity.ok(taskRepresentationModelAssembler.toModel(updatedTask));
+  }
+
+  /**
+   * This endpoint force claims a Task if possible even if it is already claimed by someone else.
+   *
+   * @param taskId the Id of the Task which should be force claimed
+   * @param userName TODO: this is currently not used
+   * @return the force claimed Task
+   * @throws TaskNotFoundException if the requested Task does not exist.
+   * @throws InvalidStateException if the state of the requested Task is not READY.
```

Comment:
if the state of Task with taskId is in {@linkplain TaskState.END_STATES}. This needs to be adjusted for the Java API aswell.

Reply:
It seems like we don't link in the REST Docs, I'll write it plain text. In the Java API I'll link of course.